### PR TITLE
fix pylint problems already on master

### PR DESCRIPTION
**Pull request by whoburg**

### CODE REVIEW

**Comment by whoburg** [comment]
test models please

**Comment by whoburg** [comment]
test models please

**Comment by whoburg** [comment]
please test models

File: gpkit/nomials/variables.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def __new__(cls, shape, *args, **descr):
             vl[i].key.arraykey = arraykey
 
         if descr.pop("newvariable", True):
-            from .. import MODELS, MODELNUMS, NAMEDVARS
+            from .. import MODELS, MODELNUMS
```
Comment (whoburg) [inline gpkit/nomials/variables.py]: @bqpd, please confirm this is the correct fix as opposed to `NAMEDVARS[tuple(MODELS), tuple(MODELNUMS)].append(self)`.

**Comment by whoburg** [comment]
bypassing review rules and merging because gpkit_Push is currently failing on master and this fixes that.

File: gpkit/nomials/variables.py (hunk 1 above)
Comment (bqpd) [inline gpkit/nomials/variables.py]: Whoops derp yes

**Comment by bqpd** [comment]
++branch name

**Pull request merged**